src/test/java/com/mathworks/ci/RunMatlabBuildBuilderTester.java

```diff
@@ -0,0 +1,117 @@
+package com.mathworks.ci;
+
+/**
+ * Copyright 2022 The MathWorks, Inc.
+ *  
+ */
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nonnull;
+import org.kohsuke.stapler.StaplerRequest;
+import hudson.EnvVars;
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.Launcher;
+import hudson.Launcher.ProcStarter;
+import hudson.model.AbstractProject;
+import hudson.model.Result;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import hudson.tasks.BuildStepDescriptor;
+import hudson.tasks.Builder;
+import net.sf.json.JSONObject;
+
+public class RunMatlabBuildBuilderTester extends RunMatlabBuildBuilder {
```

Comment:
There is a Tester for each of the types of MATLAB tasks that are almost all the same. We might be able to consolidate here going forward as well (Mock MATLAB executor for all of them?)

Reply:
+1